src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,9 @@
+@mixin displayFlex($direction, $justify) {
+  display: flex;
+  flex-direction: $direction;
+  justify-content: $justify;
+}
+
+@mixin transition($time) {
+  transition: all $time;
```

Comment:
Better pass the properties instead of duration for better performance 
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x